pkg/virt-handler/cmd-client/client.go

```diff
@@ -559,7 +560,30 @@ func (c *VirtLauncherClient) GetDomain() (*api.Domain, bool, error) {
 	}
 	return domain, exists, nil
 }
+func (c *VirtLauncherClient) GetQemuVersion() (string, error) {
+	request := &cmdv1.EmptyRequest{}
+	ctx, cancel := context.WithTimeout(context.Background(), shortTimeout)
+	defer cancel()
+
+	versionResponse, err := c.v1client.GetQemuVersion(ctx, request)
+	if versionResponse == nil {
+		return "", err
+	}
+
+	var response *cmdv1.Response
+	if versionResponse != nil {
+		response = versionResponse.Response
+	}
+
+	if err = handleError(err, "GetQemuVersion", response); err != nil {
```

Comment:
Just pass in `versionResponse.Response` here? You already checked if `versionResponse` is nil.

Reply:
Refactored a bit. Many thanks for the suggestions :)